6.x/crud-fields.md

```diff
@@ -1401,6 +1401,87 @@ Input preview:
 
 <hr>
 
+<a name="dropzone"></a>
+### dropzone <span class="badge badge-pill badge-info">PRO</span>
+
+Show a [Dropzone JS Input](https://docs.dropzone.dev/).
+
+**Step 0.** Make sure the db column can hold the amount of text this field will have. For example, for MySQL, `VARCHAR(255)` might not be enough all the time (for 3+ files), so it's better to go with `TEXT`. Make sure you're using a big column type in your migration or db.
```

Comment:
Shouldn't we also tell people to cast this DB column to `array` in their Models?

Reply:
Only required if not using the `->withFiles()`.

We don't document how to do it without `->withFiles()`, just that it can be done using model events, mutators etc. 

So if developers are writing their own upload process they should be responsible/aware of what get's save in database and what gets retrieved. Casting is optional, they can do manual casting like we do on the uploader, so **I don't think there is the need to mention it here**. 

Let me know if you disagree and I can add it there as a note. 

Cheers

Reply:
```suggestion
**Step 0.** Make sure the model attribute can hold all the information needed. Ideally, your model should cast this attribute as `array` and your migration should make the db column either `TEXT` or `JSON`. Other db column types such as `VARCHAR(255)` might not be enough all the time (for 3+ files).
```